assets/sass/components/surveys/_googlesitekit-surveys.scss

```diff
@@ -57,6 +60,10 @@
 		.mdc-text-field-helper-line {
 			padding-left: 0;
 		}
+
+		.mdc-text-field-helper-text {
+			opacity: 1;
+		}
```

Comment:
This fixes the problem but not quite in the right place.

It turns out the helper text is designed to be conditionally shown and likely uses `opacity` to avoid a layout shift.

All we need to do is pass the `persistent` prop to `HelperText` to indicate that the text is to always be shown as the component is made to also support input validation.

See https://github.com/material-components/material-components-web-react/blob/master/packages/text-field/helper-text/README.md#props

You can also see this work on `develop` by toggling the prop in React devtools.

Reply:
Huh, whoops, thanks! 👍🏻